### PR TITLE
[14.0] cetmix_tower_server Fix translation related issues

### PR DESCRIPTION
**Pull request by ivs-cetmix**
## cetmix_tower_server

- Explicit translations are used in "string" and "help" keys in fields.
- Wrong compute function name is used in computed field.

## cetmix_tower_yaml

- Update module description.

## Summary by CodeRabbit

- **New Features**
	- Updated URLs in documentation to point to the stable version `14.0`.
	- Reformatted the list of supported data models, indicating all are now available.
- **Bug Fixes**
	- Removed placeholder text and "Coming soon" sections from various documentation files.
- **Documentation**
	- Standardized formatting for export/import instructions and important considerations.
	- Added clarity to user messages regarding missing required variables in the wizard.

### CODE REVIEW

File: cetmix_tower_server/wizards/cx_tower_server_template_create_wizard.py [hunk 1]
```diff
@@ -149,6 +146,5 @@ class CxTowerServerTemplateCreateWizardVariableLine(models.TransientModel):
     variable_reference = fields.Char(related="variable_id.reference", readonly=True)
     value_char = fields.Char(string="Value")
     required = fields.Boolean(
-        string="Required",
-        help=_("Indicates if this variable is mandatory for server creation"),
+        help="Indicates if this variable is mandatory for server creation",
```
[the remark this answers is not in the document]
Comment (ivs-cetmix) [reply cetmix_tower_server/wizards/cx_tower_server_template_create_wizard.py:149]: Field keys are translated automatically, no need to translate them using "_" function.

File: cetmix_tower_server/wizards/cx_tower_server_template_create_wizard.py [hunk 2]
```diff
@@ -48,18 +48,15 @@
     line_ids = fields.One2many(
         comodel_name="cx.tower.server.template.create.wizard.line",
         inverse_name="wizard_id",
-        string=_("Configuration Variables"),
+        string="Configuration Variables",
```
[the remark this answers is not in the document]
Comment (ivs-cetmix) [reply cetmix_tower_server/wizards/cx_tower_server_template_create_wizard.py:51]: Field keys are translated automatically, no need to translate them using "_" function.

**Comment by CetmixGitDrone** [comment]
On my way to merge this fine PR!
Prepared branch [14.0-dev-ocabot-merge-pr-164-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-164-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at 2d2694e852a5f440a3b7601a9ceb64520c4f5d6a. Thanks a lot for contributing to cetmix. ❤️

**Pull request merged**